src/uct/tcp/tcp_sockcm_ep.h

```diff
@@ -18,13 +18,9 @@ typedef enum uct_tcp_sockcm_ep_state {
     UCT_TCP_SOCKCM_EP_CLIENT_CONNECTED_CB_INVOKED   = UCS_BIT(7),  /* ep invoked the connect_cb on the client side */
     UCT_TCP_SOCKCM_EP_SERVER_NOTIFY_CB_INVOKED      = UCS_BIT(8),  /* ep invoked the notify_cb on the server side */
 
-    /* a connected ep on the server side after upper layer's call to uct_ep_create */
-    UCT_TCP_SOCKCM_EP_SERVER_CONN_CREATED = UCT_TCP_SOCKCM_EP_ON_SERVER |
-                                            UCT_TCP_SOCKCM_EP_SERVER_CREATED,
-
     /* a connected ep completed the data sending */
-    UCT_TCP_SOCKCM_EP_CONN_SENT           = UCT_TCP_SOCKCM_EP_PRIV_DATA_PACKED |
-                                            UCT_TCP_SOCKCM_EP_DATA_SENT
+    UCT_TCP_SOCKCM_EP_CONN_SENT                     = UCT_TCP_SOCKCM_EP_PRIV_DATA_PACKED |
+                                                      UCT_TCP_SOCKCM_EP_DATA_SENT
```

Comment:
why need this special flag? isn't it enough to test UCT_TCP_SOCKCM_EP_DATA_SENT?